### DISCUSSION TITLE
Bug Antares 8.5/8.6 Size limit hit for file

### DISCUSSION THREAD

**Discussion by JasonMarechal25** [AntaresSimulatorTeam/antares-xpansion#690] [General]
Quand je lance les tests sous windows avec 8.5/8.6 Simulateur echoue avec l'erreur "Size limit hit for file general data"
De mémoire @a-zakir avait fixer ça pour windows dans Simulator mais on avait peut-être pas re-release une version avec le fix windows.
Je regarde ça

**Comment by a-zakir** [comment]
@JasonMarechal25 @flomnes  https://github.com/AntaresSimulatorTeam/Antares_Simulator/pull/1503 🙃

**Reply by flomnes** [reply]
Il faut utiliser une version patchée, aka 8.6.2 ou 8.7.0

**Reply by JasonMarechal25** [reply]
Bug présent en 8.6.2 pour windows
